src/Commands/CreateRLSPoliciesForTenantTables.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Stancl\Tenancy\Database\Concerns\BelongsToPrimaryModel;
+
+class CreateRLSPoliciesForTenantTables extends Command
+{
+    protected $signature = 'tenants:create-rls-policies';
+
+    public function handle(): int
+    {
+        foreach (config('tenancy.models.rls') as $modelClass) {
+            /** @var Model $model */
+            $model = new $modelClass;
+
+            $this->makeModelUseRls($model);
+        }
+
+        return Command::SUCCESS;
+    }
+
+    protected function makeModelUseRls(Model $model): void
+    {
+        $table = $model->getTable();
+        $tenantKey = tenancy()->tenantKeyColumn();
+
+        DB::transaction(fn () => DB::statement("DROP POLICY IF EXISTS {$table}_rls_policy ON {$table}"));
+
+        if (! Schema::hasColumn($table, $tenantKey)) {
+            // Table is not directly related to tenant
+            if (in_array(BelongsToPrimaryModel::class, class_uses_recursive($model::class))) {
+                $this->makeSecondaryModelUseRls($model);
+            } else {
+                $modelName = $model::class;
+
+                $this->components->info("Table '$table' is not related to tenant. Make sure $modelName uses the BelongsToPrimaryModel trait.");
+            }
+        } else {
+            DB::transaction(fn () => DB::statement("CREATE POLICY {$table}_rls_policy ON {$table} USING ({$tenantKey}::TEXT = current_user);"));
```

Comment:
What's the point of wrapping a single DB call in a transaction?

Reply:
I thought that if the call was wrapped in a transaction, and the call failed, the DB would be rolled back. That seemed like the correct thing to do – if the call would've broken something, the rollback would fix that

Reply:
I deleted the single statement transactions, and I put some method calls that execute multiple statements into transactions

https://github.com/archtechx/tenancy/compare/8ccc27e8c3d5c2ad106dcbb77321a343862e456e..61519ecec34c41efc7b9e72f782da353204ba0cf